### PR TITLE
user page - project rollover buttons, nav breadcrumb

### PR DESCRIPTION
**Pull request by imgrby**
temporary demo: hover on project card to see buttons. click "open" in project card to see nav style for project view.

### CODE REVIEW

File: browser/scripts/userpage.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-var userpageUI = new function() {
+var UserpageUI = new function() {
```
Comment (gmarinov) [inline browser/scripts/userpage.js]: that's a variable, not a class name, hence the camel-case

Comment (imgrby) [inline browser/scripts/userpage.js]: fixed

File: browser/scripts/ui.js [hunk 2]
```diff
@@ -87,78 +89,6 @@ VizorUI.prototype.onSearchResultsChange = function() {
 	}
 };
 
-VizorUI.prototype.openPublishGraphModal = function() {
```
Comment (gmarinov) [inline browser/scripts/ui.js]: i'm afraid this needs to stay in, and its call in application.js too

Comment (imgrby) [inline browser/scripts/ui.js]: it should live in account controller while all support actions are also there

**Pull request closed without merging**